.github/workflows/lint.yml

```diff
@@ -0,0 +1,34 @@
+name: Lint
+
+on: [pull_request]
+
+jobs:
+  test:
+    name: spelling checker
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checking out source code
+        uses: actions/checkout@v3
+
+      - name: Getting changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v32
+
+      - name: Printing changed files
+        run: |
+          for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
+            echo "$file was changed"
+          done
+
+      - name: Installing codespell
+        run: |
+          pip install codespell==2.2.1
+
+#      # lint all files (test purpose)
```

Comment:
I think we could remove these test code.

Reply:
Actually, these are not test codes. I prefer to keep this as we will switch to this someday as no more old existing misspellings. (Don't have to figure it out again)

Reply:
I saw you have removed these code. 

It makes our script more clean, and we can still keep that test code for future purposes.